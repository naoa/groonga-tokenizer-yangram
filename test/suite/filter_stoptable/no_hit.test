register tokenizers/yangram

table_create @yangram_stopwords TABLE_HASH_KEY ShortText

load --table @yangram_stopwords
[
{"_key": "that"},
{"_key": "He"}
]

tokenize TokenYaBigram "This is a pen" NormalizerAuto
