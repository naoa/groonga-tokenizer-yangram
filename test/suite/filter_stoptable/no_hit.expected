register tokenizers/yangram
[[0,0.0,0.0],true]
table_create @yangram_stopwords TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table @yangram_stopwords
[
{"_key": "that"},
{"_key": "He"}
]
[[0,0.0,0.0],2]
tokenize TokenYaBigram "This is a pen" NormalizerAuto
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "this",
      "position": 0
    },
    {
      "value": "is",
      "position": 1
    },
    {
      "value": "a",
      "position": 2
    },
    {
      "value": "pen",
      "position": 3
    }
  ]
]
